atc/atccmd/command.go

```diff
@@ -124,6 +125,7 @@ type RunCommand struct {
 	InterceptIdleTimeout time.Duration `long:"intercept-idle-timeout" default:"0m" description:"Length of time for a intercepted session to be idle before terminating."`
 
 	EnableGlobalResources bool `long:"enable-global-resources" description:"Enable equivalent resources across pipelines and teams to share a single version history."`
+	EnableZstdCompression bool `long:"enable-zstd-compression" description:"Enable zstd compression for internal streaming."`
```

Comment:
Can we change this to something like `--streaming-artifacts-compression=[gzip|zstd]` with a default of `gzip`? Feels more future-proof if we want to experiment with various compression algorithms.